.github/workflows/ci.yml

```
@@ -18,4 +18,15 @@ jobs:
 
       - name: build docker image
         run: |
-          docker build . -t ${{ env.docker_image }}:temp
\ No newline at end of file
+          docker build . -t ${{ env.docker_image }}:temp
+  
+  build_binary:    
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)